src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -307,7 +275,11 @@ Here's what's happening behind the scenes:
 
 * Session replay works by rebuilding the user's DOM, which includes your website's structure and visual elements. To achieve this, our agent needs to access your CSS files to accurately capture styling information. We then inline these styles directly into the replay recording, eliminating the need to reference the original CSS files during playback.
 
-* If your CSS files are hosted on a separate domain, CORS may prevent our agent from accessing them. This can result in incomplete or inaccurate replays, as essential styling information may be missing.
+* If your CSS files are hosted on a separate domain, CORS may prevent our agent from accessing them.
+
+* The agent attempts to collect all CSS, but if it can't, it sends the replay with broken styles along with attributes `hasCrossOriginCSS:true` and `inlinedCrossOriginCSS:false` so the session replay UI knows some styles are broken and provides a warning message with a link to a fix. The agent is set up to report:
+    * `invalidStylesheetsDetected` will resolve to true if we find cross-origin stylesheets
+    * `inlinedAllStylesheets` will resolve to false if any stylesheets weren’t able to be inlined by the agent. If false, you'll see a message in the UI to fix it.
```

Comment:
The SessionReplay events are not visible to customers so I don't know why we would tell them attributes on the event.

Reply:
Thanks for this callout. @CherylFrankenfield , if customers cannot see sessionreplay events, I think I should remove these bullet points. Do you have any objections?